libs/blocks/faas-config/faas-config.js

```diff
@@ -24,6 +24,8 @@ const sortObjects = (obj) => Object.entries(obj).sort((a, b) => {
   return x < y ? -1 : x > y ? 1 : 0;
 });
 
+const cloneObj = (obj) => JSON.parse(JSON.stringify(obj));
```

Comment:
maybe `ogState` or `initialState`? `cloneObj`. 

I see you use this function with defaultState below, and you are stringifying and parsing, perhaps could just do that on the default state?

Reply:
Renaming to `deepCopy`, this is the fastest way to do a deep copy and I don't want to couple the function name to the default state since this could be used for different cases.